{{cookiecutter.project_shortname}}/invenio.cfg

```diff
@@ -1,15 +1,54 @@
-ACCOUNTS_SESSION_REDIS_URL="redis://localhost:6379/1"
-BROKER_URL="amqp://guest:guest@localhost:5672/"
-CACHE_REDIS_URL="redis://localhost:6379/0"
-CACHE_TYPE="redis"
-CELERY_BROKER_URL="amqp://guest:guest@localhost:5672/"
-CELERY_RESULT_BACKEND="redis://localhost:6379/2"
-SEARCH_ELASTIC_HOSTS=[{"host": "localhost", "port": 9200}]
```

Comment:
Removed ones are in invenio-app-rdm anyway.

Reply:
Minor: I kind of consider the "endpoints" of this services are super likely to be customized, no?

Reply:
Yes, that's another aspect we didn't talk about much: we still have to think about the dev vs real prod configuration story. If the hoster modifies the `invenio.cfg` for production, then running it locally (dev) will probably not work anymore (invalid service hosts for instance). There should be a clear story here which could be documented as:

- Modify `invenio.cfg` when it affects your production and development environment
- Modify `.production.env` when it affects only your production environment

and then we have invenio-cli place that `.production.env` at the right place. Or something similar. I am making an issue for that so we can deal with it separately.

Reply:
Task: https://github.com/inveniosoftware/invenio-cli/issues/52